docs/pages/docs/docs-theme/google-analytics.mdx

```diff
@@ -0,0 +1,65 @@
+import { Steps } from 'nextra/components'
+
+# Google Analytics
+
+This page describes how to configure your Nextra website with Google Analytics (`gtag.js`).
+
+**Note:** This guide assumes you already have a [Google Analytics measurement id](https://support.google.com/analytics/answer/9304153) for your Nextra site.
+
+## Configuration
+
+<Steps>
+
+### Create a Custom App
+
+Add a custom App (`_app.mdx`) to the `pages` directory of your project.
+
+### Add Google Analytics to Your App
+
+Use [`next/script`](https://nextjs.org/docs/pages/building-your-application/optimizing/scripts) to add your Google Analytics tracking code to the custom App (`pages/_app.mdx`) as follows:
+
+```jsx filename="pages/_app.mdx"
```

Comment:
Seems your example will not fire when the page is changed with client-side rendering, here is official example https://github.com/vercel/next.js/blob/canary/examples/with-google-analytics/pages/_app.js

Reply:
I've updated the doc with examples that should work with client-side rendering.